io/plugins/eu.esdihumboldt.hale.io.jdbc/schema/eu.esdihumboldt.hale.io.jdbc.config.exsd

```diff
@@ -176,6 +176,30 @@
                </appinfo>
             </annotation>
          </attribute>
+         <attribute name="isSchemaSelectionEnable" type="boolean" use="required">
```

Comment:
I would prefer having a child element for `driver` that encompasses all the configuration related to schema selection. Then the attribute names can also be more concise.
In the extension it might look like this:
```
<driver ...>
  ...
  <schema-selection enabled="true" multiple="false" selector="xxx.Class" />
</driver>
```